dashboard/client/src/api/functionsApi.js

```diff
@@ -53,7 +53,7 @@ class FunctionsApi {
         endpoint = this.baseURL + '/function/' + item.name;
       }
 
-      let shortSha = item.labels['Git-SHA'];
+      let shortSha = item.labels['com.openfaas.cloud.git-SHA'];
```

Comment:
I'm going to guess we should make `SHA` lower too so that it matches everything else?

Reply:
Ok